### PR TITLE
Fix release notes in marketplace

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
1. Fixed release notes in the marketplace for packs 'CommonTypes', 'MicrosoftTeams' and 'PrismaSaaSSecurity'.
2. Removed the usage in the modified_files_data.
3. Fixed the filtering for modified release notes files to filter by display name in id set as well.

### CODE REVIEW

**Comment by yaakovpraisler** [comment]
Failed test: https://app.circleci.com/pipelines/github/demisto/content/199509/workflows/087536ae-6c8f-4d55-8fa5-fae495eac9a3/jobs/584554

**Comment by yaakovpraisler** [comment]
Successful upload: https://code.pan.run/xsoar/content/-/pipelines/3900378

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -1794,26 +1761,37 @@ def filter_entries_by_display_name(release_notes: dict, display_names: list, rn_
...
+        for rn_header in release_notes:
+            filtered_entries: dict = {}
 
-            logging.debug(f"Searching display name '{display_name}' in '{display_names}'.")
-            # TODO: The third condition should be removed after the refactoring. (Also the function)
-            if display_name != '[special_msg]' and display_name.replace("New: ", "") not in display_names \
-                    and not get_id_set_entity_by_display_name(display_name.replace("New: ", ""),
-                                                              pack_folder, id_set):
-                continue
+            for display_name, rn_entry in release_notes[rn_header].items():
```
Comment (dantavori) [inline Tests/Marketplace/marketplace_services.py:1768]:
```suggestion
            for content_item_display_name, content_item_rn_notes in content_type_rn_entries.items():
```

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -1794,26 +1761,37 @@ def filter_entries_by_display_name(release_notes: dict, display_names: list, rn_
...
+
+                logging.debug(f"Searching display name '{display_name}' with rn header '{rn_header}' in in id set.")
+                if display_name != '[special_msg]' and not get_id_set_entity_by_display_name(
```
Comment (dantavori) [inline Tests/Marketplace/marketplace_services.py:1771]:
```suggestion
                if display_name != '[special_msg]' and not is_content_item_in_id_set(
```

File: Packs/CommonTypes/ReleaseNotes/3_3_22.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 
 #### Indicator Fields
-- **Key Values**
+- **Key Value**
```
Comment (dantavori) [inline Packs/CommonTypes/ReleaseNotes/3_3_22.md:3]: did we make the changes on these RN files on purpose?

**Review by dantavori — CHANGES_REQUESTED** [review]
looks good to me, @Noy-Maimon can you please go over this too?

File: Tests/Marketplace/marketplace_services.py [hunk 4]
```diff
@@ -788,7 +788,6 @@ def _create_changelog_entry(self, release_notes, version_display_name, build_num
             return self.filter_changelog_entries(
                 entry_result,
                 version_display_name,
-                modified_files_data,
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Tests/Marketplace/marketplace_services.py:791]: done

File: Tests/Marketplace/marketplace_services.py [hunk 5]
```diff
@@ -1794,26 +1761,37 @@ def filter_entries_by_display_name(release_notes: dict, display_names: list, rn_
         Returns:
             (dict) The filtered release notes entries.
         """
-        filtered_entries: dict = {}
-        for display_name, rn_entry in release_notes[rn_header].items():
+        filtered_release_notes: dict = {}
+        for rn_header in release_notes:
```
[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Tests/Marketplace/marketplace_services.py]: Done all comments

**Comment by yaakovpraisler** [comment]
Successafull upload where release notes are generated correctly on XSOAR and on marketplacev2.
https://code.pan.run/xsoar/content/-/pipelines/3941518

**Review by dantavori — APPROVED** [review]

**Pull request merged**